mysql-test/suite/sys_vars/r/replicate_rewrite_db.result

```diff
@@ -84,5 +84,16 @@ db1->db2,db3->db4
 SELECT * FROM INFORMATION_SCHEMA.GLOBAL_VARIABLES WHERE VARIABLE_NAME='replicate_rewrite_db';
 VARIABLE_NAME	VARIABLE_VALUE
 REPLICATE_REWRITE_DB	db1->db2,db3->db4
+# Check restart with appending the value
+# restart: --replicate_rewrite_db=X->Y
+SELECT @@GLOBAL.replicate_rewrite_db;
+@@GLOBAL.replicate_rewrite_db
+test->rewrite,mysqltest1->test,a->b,X->Y
+# Check restart with wrong value on CLI
+2022-09-09 11:12:40 0 [ERROR] Bad syntax in replicate-rewrite-db
```

Comment:
The timestamp here will be problematic on replay. instead of using `cat_file`, I suggest to use `source include/assert_grep.inc`, e.g.

```
--let assert_text= Invalid replicate_rewrite_db CLI argument should produce error
--let assert_count= 1
--let assert_select= [ERROR] Bad syntax in replicate-rewrite-db
--let assert_file=$MYSQL_TMP_DIR/mysqld--replicate.err
--source include/assert_grep.inc
```

Reply:
Done with `--replace_regex /mysqld/mariadbd/ /\d\d\d\d-\d*-\d* *\d*:\d*:\d* \d* //`